python/spear/config/default_config.vehicle.yaml

```diff
@@ -0,0 +1,46 @@
+#
+# Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+#
+
+VEHICLE:
+  VEHICLE_PAWN:
+    SKELETAL_MESH: "/Vehicle/OpenBot/Meshes/OpenBot_Mesh.OpenBot_Mesh"
+    ANIM_INSTANCE: "/Vehicle/OpenBot/Meshes/OpenBot_Animation.OpenBot_Animation_C"
+
+    CAMERA_COMPONENT:
+      POSITION_X: -60.0            # Camera relative position in [cm]
+      POSITION_Y: 0.0              # Camera relative position in [cm]
+      POSITION_Z: 30.0             # Camera relative position in [cm]
+      PITCH: -10.0                 # Camera relative orientation in [°]
+      YAW: 0.0                     # Camera relative orientation in [°]
+      ROLL: 0.0                    # Camera relative orientation in [°]
+      FOV: 70.0                    # Camera field of view in [°]
+    
+    IMU_COMPONENT:
+      POSITION_X: 8.0              # IMU relative position in [cm]
+      POSITION_Y: 0.0              # IMU relative position in [cm]
+      POSITION_Z: 10.0             # IMU relative position in [cm]
+      PITCH: 0.0                   # IMU relative orientation in [°]
+      YAW: 0.0                     # IMU relative orientation in [°]
+      ROLL: 0.0                    # IMU relative orientation in [°]
+
+  VEHICLE_COMPONENT:
+    MASS: 11.2                     # Mass of the vehicle chassis in [kg]
+    DRAG_COEFFICIENT: 1.0          # DragCoefficient of the vehicle chassis
+    CHASSIS_WIDTH: 15.0            # Chassis width used for drag force computation in [cm]
+    CHASSIS_HEIGHT: 15.0           # Chassis height used for drag force computation in [cm]
+    ENABLE_SUSPENSION: True
+    ENABLE_WHEEL_FRICTION: True
+    ENABLE_LEGACY_WHEEL_FRICTION: False
+
+  VEHICLE_WHEEL:
+    WHEEL_MASS: 0.033                 # Mass of this wheel in [kg]
+    WHEEL_RADIUS: 3.3                 # Radius of the wheel in [cm]
+    WHEEL_WIDTH: 2.5                  # Width of the wheel in [cm]
+    FRICTION_FORCE_MULTIPLIER: 3.0
+    SUSPENSION_MAX_RAISE: 0.001       # How far the wheel can go above the resting position in [m]
+    SUSPENSION_MAX_DROP: 0.001        # How far the wheel can drop below the resting position in [m]
+    SUSPENSION_DAMPING_RATIO: 0.5     # The rate at which energy is dissipated from the spring. Standard cars have values between 0.8 and 1.2. Values < 1
+                                      # are more sluggish, values > 1 or more twitchy.
```

Comment:
Or more twitchy? Do you mean are more twitchy? Can we add comments from the Unreal docs to any of these, e.g., `FRICTION_FORCE_MULTIPLIER`?

Reply:
Corrected the comment.
I couldn't find any comments on `FRICTION_FORCE_MULTIPLIER` in Unreal docs, so didn't add any here